src/constants/translations/en/common.json

```diff
@@ -69,7 +69,8 @@
   "errorMessages": {
     "emailValid": "Email should be of the following format: “local-part@domain.com”",
     "passwordLength": "Password cannot be shorter than 8 and longer than 25 characters",
-    "passwordValid": "Password must contain at least one alphabetic and one numeric character",
+    "passwordAlphabeticAndNumeric": "Password must contain at least one alphabetic and one numeric character",
+    "passwordValidSymbols": "Password must contain only valid symbols",
```

Comment:
As a user, I can't know what is a 'valid symbol' here.
Could it be better to use something like `Password must contain only alphabetic and numeric characters`?